qiskit_ibm_runtime/options.py

```diff
@@ -191,6 +198,8 @@ class Options:
               Default: ``True``.
     """
 
+    max_execution_time: int = None
+    job_tags: List[str] = None
```

Comment:
I created a new `EnvironmentOptions` in my last PR as an attempt to organize less commonly used options (such as `image`) that is not related to transpilation or execution. I think `job_tags` can go there too, but `max_execution_time` can stay at level 1 as it's more likely to be used.